tests/admin_changelist/tests.py

```diff
@@ -1686,3 +1686,72 @@ def test_save_without_changes_warns_on_pending_action(self):
             )
         finally:
             alert.dismiss()
+
+
+@modify_settings(MIDDLEWARE={'remove': 'django.contrib.admin.tests.CSPMiddleware'})
```

Comment:
Is this OK? I removed it because I was getting:
![image](https://user-images.githubusercontent.com/16822952/150038046-d0bef074-cd27-4517-be52-691d6b46c827.png)